.env.dist

```diff
@@ -0,0 +1,56 @@
+APP_NAME=Laravel
+APP_ENV=local
+APP_KEY=base64:3R7Tumc/H0dz8qwMa0rGdxoSDGOFZgcT3iKmEkDJQ8E=
+APP_DEBUG=true
+APP_URL=http://meetup
```

Comment:
In `readme.md` file, you wrote that address where the environment is available is `http://localhost/`, but here we have `http://meetup`. Please be more consistent.